teams/praha.yml

```diff
@@ -3,11 +3,19 @@
   role: Organizátorka/Koučka
   links:
   - mail-link: mailto:valkova.zuza@seznam.cz
-- name: Iveta Jelenová
-  img: img/praha/team/ivet.jpg
+- name: Lucka Říhová
+  img: img/praha/team/lucka.jpg
   role: Organizátorka - kurz NTK
   links:
-  - mail-link: mailto:iveta@jelen.biz
+  - mail-link: mailto: luc.rihova@gmail.com
```

Comment:
Nedávej za mailto: mezeru prosím, nemá tam byt, a navíc to tu může dělat problém v yamlu